client/components/pack/AddPack.js

```diff
@@ -37,10 +40,35 @@ export const AddPack = () => {
    * @return {void}
    */
   const handleAddPack = () => {
-    dispatch(addPack({ name, owner_id: user?._id }));
-    setName('');
+    try {
+      addPackValidation.parse({ name, owner_id: user?._id });
+      dispatch(addPack({ name, owner_id: user?._id }));
+      setName('');
+      setFormErrors({});
+    } catch (error) {
+      const errorObject = JSON.parse(error.message);
```

Comment:
i feel like we repeat this a lot, should we move to helper or form utility?

Reply:
For the implementation that’s not a useful piece of code

Reply:
From old pr as I live forward I’m taking them out